.github/workflows/ci-schedule-compatibility.yaml

```diff
@@ -13,18 +13,18 @@ jobs:
     strategy:
       matrix:
         kubeapiserver-version: [ v1.21.10, v1.22.7, v1.23.4, v1.24.2, v1.25.0, v1.26.0, v1.27.3, v1.28.0 ]
-        karmada-version: [ release-1.4, release-1.5, release-1.6 ]
+        karmada-version: [ release-1.5, release-1.6, release-1.7 ]
 
         include:
-        - karmada-version: release-1.4
-          go-version: 1.19.5
-
         - karmada-version: release-1.5
           go-version: 1.19.5
 
         - karmada-version: release-1.6
           go-version: 1.20.5
 
+        - karmada-version: release-1.7
+          go-version: 1.20.5
```

Comment:
Actually we use Go@1.20.6 for release-1.7.
It's not a big deal, but it would be great to keep the same version of release 1.7.